src/service/viewer-impl.js

```diff
@@ -221,6 +221,10 @@ export class Viewer {
     this.performanceTracking_ = this.params_['csi'] === '1';
     dev.fine(TAG_, '- performanceTracking:', this.performanceTracking_);
 
+    this.shareTrackingIncomingFragment_ = this.params_['share-tracking'];
```

Comment:
I know that. I won't read from the url because the url the viewer generated cannot contain the share-tracking info (because the when the viewer generate the url, it doesn't know the share-tracking id). I won't read it from the url anymore because it's not there. I will remove this part of the code.